app/assets/javascripts/application_revised.js

```diff
@@ -11,7 +11,7 @@
 //= require mustache
 //= require Stickyfill
 //= require select2/select2.full.js
-//= require selectize/standalone/selectize.js
+//= require external_scripts/selectize.js
 //= require external_scripts/selectize_placeholder_plugin.js
```

Comment:
Selectize is broken 

<img width="438" alt="Screen Shot 2019-11-22 at 5 46 42 PM" src="https://user-images.githubusercontent.com/4433943/69465849-556d6980-0d50-11ea-86cb-177328f8eba8.png">
<img width="505" alt="Screen Shot 2019-11-22 at 5 46 52 PM" src="https://user-images.githubusercontent.com/4433943/69465850-556d6980-0d50-11ea-99a0-e2c204d96676.png">

Reply:
Does it make sense to switch to webpack only scss at this point?

We still will need sprockets for the email scss...

To make the switch, we'd need to bump the version of bootstrap that the main app uses. This would take a bunch of work, but might be worth it? (see #1005)

Reply:
My hunch is that it makes sense to err on the side of incrementalism and finish migrating to Webpack once Rails is upgraded, but I'm not totally set on that yet.

From what I'm seeing so far the Rails update may compel tackling the [Bootstrap upgrade](https://github.com/bikeindex/bike_index/pull/1429) anyway — there are breaking changes in the way assets are sourced that lead to a lot of undefined mixins / variables, which breaks asset compilation. 

My plan has been to update Sprockets, then Bootstrap, then Rails, in that order. I'm updating the second step to "handle" Bootstap, where "handle" is one of the following approaches:

1) Minimal: Figure out how to accommodate the changes in Sprockets so assets compile as they do now, without changes to Bootstrap (which may or may not be possible w/o forking)
2) Expedient: Temporarily vendorize Bootstrap stylesheets to fix asset compilation and return to the Bootstrap upgrade later
3) Thorough: Update Bootstrap, which might require backing out some customizations and [incrementally migrating](https://getbootstrap.com/docs/4.0/migration/) (potentially time-consuming)
4) Aggressive: Lean into the already partially completed migration from Sprockets to Webpack (risky, but also potentially time-saving? I think so at least, but I'd need to explore this a bit more to know exactly how.)

There's a lot of wilderness in our assets directory — are the Bootstrap files there in fact customized? Or just vendorized for some other reason?

Reply:
As per my other comment - I think option 2 is a good choice here, so long as there are comments explaining what is happening and why.